Lib/idlelib/idle_test/test_searchbase.py

```diff
@@ -48,7 +48,6 @@ def test_open_and_close(self):
         self.dialog.default_command = None
 
         toplevel = Toplevel(self.root)
-        self.addCleanup(toplevel.destroy)
```

Comment:
def cleanup():
    toplevel.update()
    toplevel.destroy()
self.addCleanup(cleanup)

Reply:
See my comment on the cli issue.